src/ts/components/button.ts

```diff
@@ -1,15 +1,16 @@
 import {ComponentConfig, Component} from './component';
 import {DOM} from '../dom';
 import {EventDispatcher, NoArgs, Event} from '../eventdispatcher';
+import {LocalizableText , i18n} from '../localization/i18n';
```

Comment:
why do we have spaces in this import when we do not have any of the imports above?

Reply:
We want to change the style to this but we don't want to change the whole codebase at once.

Reply:
imo. that kind of changes should be handled in an seperate issue. changing the code style part by part causes more inconsistency. 
I have already created an [issue ](https://github.com/bitmovin/player-issues-web/issues/2523) for such things, maybe add it as a comment there?

Reply:
This is simply the way how we have handled this until today. This is an open source repo so this may seem odd but we want to keep changes (and merge conflicts in forks) to a minimum.

Reply:
updated